typing/ctype.ml

```diff
@@ -1778,11 +1795,12 @@ let instance_prim_mode (desc : Primitive.description) ty =
   let is_poly = function Primitive.Prim_poly, _ -> true | _ -> false in
   if is_poly desc.prim_native_repr_res ||
        List.exists is_poly desc.prim_native_repr_args then
-    let mode = Locality.newvar () in
-    let finalret = prim_mode (Some mode) desc.prim_native_repr_res in
+    let mode_l = Locality.newvar () in
+    let mode_y = Yielding.newvar () in
+    let finalret = prim_mode (Some mode_l) (Some mode_y) desc.prim_native_repr_res in
     instance_prim_locals desc.prim_native_repr_args
-      mode (Alloc.disallow_right Alloc.legacy) finalret ty,
-    Some mode
+      mode_l mode_y (Alloc.disallow_right Alloc.legacy) finalret ty,
+    Some mode_l
```

Comment:
Here we are dropping `mode_y`, which is unsafe if we allow users to write `global yielding` and `local unyielding` on primitives. (This `instance_prim_mode` is used in `includecore.ml` for value inclusion)

Reply:
Will fix.

Reply:
Fixed. I'm not very concerned about unsoundness here (regarding yieldingness, not locality), but can add tests to be sure.